lib/puppet.rb

```diff
@@ -74,7 +74,7 @@ class << self
       end
 
       if locale_path
-        if Gem.loaded_specs['gettext-setup'].version < Gem::Version.new('0.8')
+        if Gem::Version.new(GettextSetup::VERSION) < Gem::Version.new('0.8')
```

Comment:
`GettextSetup::VERSION` won't exist in version 0.8. I wonder if there are any other ways to determine this. Any ideas, given the problem above with `loaded_specs`? Perhaps we could get the version number from the load path, though that sounds messy.